### ISSUE TITLE
bigger bbox sizes

### ISSUE THREAD

**Issue by onurbarut** [thtrieu/darkflow#526]
hey all,

I've trained /test/training data + 1 iphone and 1 other photo of people with tiny-yolo-4c.cfg from pre-trained tiny-yolo-voc.weights. Then I observe that the predicted bboxes are well centered however much bigger than the ground truth, even though the ave loss is around .06xxx. Is there a bug in the code about the width and height issues? anybody encountered the same??

I actually train customized yolo version for my own dataset, I also see the same issue there. I wanted to check if it also occurs with regular photos, and the result is the same. Any help?

Thanks.

my training data results:
![3](https://user-images.githubusercontent.com/18086339/34916120-bde04410-f943-11e7-8515-4606eb6aebae.jpeg)
![4](https://user-images.githubusercontent.com/18086339/34916121-c2208ec2-f943-11e7-8f2c-681c668bd256.jpeg)
tiny-yolo-voc-cfg &  tiny-yolo-voc.weights result:
![4](https://user-images.githubusercontent.com/18086339/34916134-faabf6c8-f943-11e7-8028-617ba20a6a35.jpeg)

**Comment by Yuan999999** [comment]
I met similar problems. I train ~2000 pcitures, with more than 40,000
steps. the loss is reduced to ~0.02.

However, the bounding box is very big. sometimes just cover the full
images.

it would be really appreciated if anybody has any clue!

/Yuan

2018-01-14 14:00 GMT+01:00 obarut <notifications@github.com>:

**Comment by onurbarut** [comment]
Hi all,

I've noticed that I missed the lines after 58 to change as mentioned in #482 . Making all the changes helped. Issue resolved.
![3](https://user-images.githubusercontent.com/18086339/34916520-a8b6799a-f94a-11e7-8c4c-9e241fd9e732.jpeg)

![4](https://user-images.githubusercontent.com/18086339/34916523-b0a6c65a-f94a-11e7-8ddc-e73522b59110.jpeg)
